fili-wikipedia-example/src/main/java/com/yahoo/wiki/webservice/data/config/names/WikiApiDimensionConfigInfo.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.wiki.webservice.data.config.names;
+
+import com.yahoo.bard.webservice.data.config.names.DimensionName;
+import com.yahoo.bard.webservice.util.EnumUtils;
+
+/**
+ * Hold all the Wikipedia API dimension names.
+ */
+public enum WikiApiDimensionConfigInfo implements DimensionName {
+    PAGE("Page is a document that is suitable for World Wide Web and web browsers", "wiki page", "General"),
+    LANGUAGE("Language used to write the wiki page", "wiki language", "General"),
+    USER("User is a person who generally use or own wiki services", "wiki user", "General"),
+    UNPATROLLED("Unpatrolled are class of pages that are not been patrolled", "wiki unpatrolled", "General"),
+    NEW_PAGE("New Page is the first page that is created in wiki ", "wiki newPage", "General"),
+    ROBOT("Robot is an tool that carries out repetitive and mundane tasks", "wiki robot", "General"),
+    ANONYMOUS("Anonymous are individual or entity whose identity is unknown", "wiki anonymous", "General"),
+    NAMESPACE("Namespace is a set of wiki pages that begins with a reserved word", "wiki namespace", "General"),
+    CONTINENT("Continent refers to one of the seven region", "wiki continent", "General"),
+    COUNTRY("Country is identified as a distinct national entity in political geography", "wiki country", "General"),
+    REGION("Regions are areas that are broadly divided by several distinct characteristics ", "wiki region", "General"),
+    CITY("City is a large and permanent human settlement", "wiki city", "General");
```

Comment:
No need to change them, since they are accurate, but generally descriptions of dimensions are likely to connect to the domain of the facts for which they are dimensions (usually). For example, while a city _is_ where humans live, in this context it is the city in which the user making the edit was detected to be located. (ie. it's where the user was when they made the edit).